cmd/manager/main.go

```diff
@@ -9,7 +9,7 @@ import (
 	"runtime"
 	"strings"
 
-	api "github.com/codeready-toolchain/api/pkg/apis"
+	api "github.com/codeready-toolchain/api/api/v1alpha1"
```

Comment:
same comment as in https://github.com/codeready-toolchain/toolchain-common/pull/177#discussion_r636104231 regarding the alias for the `github.com/codeready-toolchain/api/api/v1alpha1` package

Reply:
I already did that locally yesterday, but forgot to push the changes to GH: b8f6c51